sshd_config.5

```diff
@@ -969,6 +969,16 @@ or
 .Cm no .
 .Cm ChallengeResponseAuthentication
 is a deprecated alias for this.
+.It Cm KbdInteractiveLegacyInstructions
```

Comment:
What clients would break without this option? AFAIK OpenSSH clients should handle it ok (modulo formatting bugs). Are you aware of any others that have problems? I'd rather not add an option without a clearly identified need.

Reply:
No, all the ones I've tested (not many, but at least 3-4 including putty and some old ones) had not issues with it.

The reason why the option has been added was this comment: https://bugzilla.mindrot.org/show_bug.cgi?id=2876#c10

And since it was one of the blocker, I thought was somewhat needed.

However, if this is not a concern, I can definitely simplify things by avoiding that.